pkg/drivers/hyperkit/network.go

```diff
@@ -120,12 +120,11 @@ func trimMacAddress(rawUUID string) string {
 }
 
 func GetNetAddr() (net.IP, error) {
-	_, err := os.Stat(CONFIG_PLIST + ".plist")
-	if err != nil {
-		return nil, fmt.Errorf("Does not exist %s", CONFIG_PLIST+".plist")
+	plistPath := VMNetDomain + ".plist"
```

Comment:
So it this not configurable anymore?

Reply:
Still configurable - the variables were just renamed to fit the style guide: https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps